cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc

```diff
@@ -17,38 +17,50 @@ toc::[]
 //   - Chris Kang
 // ---
 
-The AWS Secrets and Configuration Provider (ASCP) provides a way to expose AWS Secrets as Kubernetes storage volumes. With the ASCP, you can store and manage your secrets in Secrets Manager and then retrieve them through your workloads running on ROSA.
-
-This is made even easier and more secure through the use of AWS STS and Kubernetes PodIdentity.
+The AWS Secrets and Configuration Provider (ASCP) provides a way to expose AWS Secrets as Kubernetes storage volumes. With the ASCP, you can store and manage your secrets in Secrets Manager and then retrieve them through your workloads running on {product-title} (ROSA).
 
 [id="cloud-experts-aws-secret-manager-prerequisites"]
 == Prerequisites
 
+Ensure that you have the following resources and tools before starting this process:
+
 * A ROSA cluster deployed with STS
 * Helm 3
-* aws CLI
-* oc CLI
-* jq
+* `aws` CLI
+* `oc` CLI
+* `jq` CLI
 
+[discrete]
 [id="cloud-experts-aws-secret-manager-preparing-environment"]
-== Preparing Environment
+=== Additional environment requirements
 
-. Validate that your cluster has STS:
+. Log in to your ROSA cluster by running the following command:
++
+[source,terminal]
+----
+$ oc login --token=<your-token> --server=<your-server-url>
+----
++
+You can find your log in token by accessing your cluster in {cluster-manager-url-pull}.
+
+. Validate that your cluster has STS by running the following command:
 +
 [source,terminal]
 ----
 $ oc get authentication.config.openshift.io cluster -o json \
   | jq .spec.serviceAccountIssuer
 ----
 +
-You should see something like the following, if not you should not proceed, instead look to the link:https://docs.openshift.com/rosa/rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.html[Red Hat documentation on creating an STS cluster].
+You should see the following:
 +
```

Comment:
```suggestion
.Example output
```
- Per [the IBM word usage guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s), avoid "should" and rewrite in a more direct way. But in any case, I suggest that you delete this sentence entirely and replace it with an `.Example output` heading.
- Per [the IBM word usage guidelines](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f), for introductory phrases, always use "following" as an adjective followed by a noun.